### ISSUE TITLE
Accessibility Improvements ‣ Product Page ‣ Image HTML

### ISSUE THREAD

**Issue by barryhughes** [woothemes/woocommerce#37603]
As originally noted [over here](https://github.com/woocommerce/woocommerce/issues/24185#issuecomment-873422417), the choice of elements in the HTML surrounding the single product image can be problematic for screen readers.

<div align="center"><img width="650" alt="product-photo-html" src="https://user-images.githubusercontent.com/3594411/230448602-8d65e072-3eb1-4159-bc7c-560e5af0688e.png"></div>

This leads to a screen reader reading:

- “Clickable figure clickable link image img-ratione.png ..."
- “Clickable” is read twice for one link.

There has been a consultation with other A11y experts on this topic to learn whether this is a violation, and it’s not. But it’s not a very elegant solution and, if possible, we should explore removal of `<figure>`, replacing it with a meaningless HTML-element like `<div>`.
